docs/user/security/audit-logging.asciidoc

```diff
@@ -3,34 +3,169 @@
+by cluster-wide privileges. For more information on enabling audit logging in
+{es}, refer to {ref}/auditing.html[Auditing security events].
 
 [IMPORTANT]
 ============================================================================
-Audit logs are **disabled** by default. To enable this functionality, you
-must set `xpack.security.audit.enabled` to `true` in `kibana.yml`.
+Audit logs are **disabled** by default. To enable this functionality, you must
+set `xpack.security.audit.enabled` to `true` in `kibana.yml`.
 ============================================================================
 
-Audit logging uses the standard {kib} logging output, which can be configured
-in the `kibana.yml` and is discussed in <<settings>>.
+The current version of the audit logger uses the standard {kib} logging output,
+which can be configured in `kibana.yml`. For more information, refer to <<settings>>.
+The audit logger uses a separate logger and can be configured using
+the options in <<audit-logging-settings>>. 
 
 ==== Audit event types
 
-When you are auditing security events, each request can generate 
-multiple audit events. The following is a list of the events that can be generated:
+When you are auditing security events, each request can generate multiple audit
+events. The following is a list of the events that can be generated:
 
 |======
 | `saved_objects_authorization_success`    | Logged when a user is authorized to access a saved
                                              objects when using a role with <<kibana-privileges>>
 | `saved_objects_authorization_failure`    | Logged when a user isn't authorized to access a saved
                                              objects when using a role with <<kibana-privileges>>
 |======
+
+==== ECS audit events
+
+beta[]
+
+[IMPORTANT]
+============================================================================
+The following events are only logged if the audit logger is enabled.
+For information on how to configure `xpack.security.audit.appender`, refer to
+<<audit-logging-settings>>.
+============================================================================
+
+Refer to the table of events that can be logged for auditing purposes. 
+
+Each event is broken down into `category`, `type`, `action` and `outcome` fields
+to make it easy to filter, query and aggregate the resulting logs. 
+
+[NOTE]
+============================================================================
+To ensure that a record of every operation is persisted even in case of an
+unexpected error, asynchronous write operations are logged immediately after all
+authorization checks have passed, but before the response from {es} is received.
+Refer to the corresponding {es} logs for potential write errors.
+============================================================================
+
+[cols="5*<",options="header"]
+|======
+| Category
+| Type
+| Action
+| Outcome
+| Description
```

Comment:
Yeh, I've noticed that too and tried to think of alternatives but they all make it harder to understand that the `action` field is actually not the most important one anymore and you're probably better off filtering/aggregating by category/type which is why I placed them first. 

I'd prefer not to split these up into multiple tables that need to be cross referenced. 

One idea I had was too add headings to break them up by category, e.g.:

```markdown
# ECS audit events

## Authentication events

Action | Outcome | Description

## Database events

### Creation events

Action | Outcome | Description

### Change events

Action | Outcome | Description

## Web events

Action | Outcome | Description
```

What do you think?

Reply:
Sure that could work. You can make one table with merged cells for the headings so that the column widths are consistent, e.g., 
```asciidoc
[cols="3*<"]
|======

3+a| ### Category: authentication

| Action
| Outcome
| Description

| `user_login`
| `success`
| User has logged in successfully.

| `user_login`
| `failure`
| Failed login attempt (e.g. due to invalid credentials).

3+a| ### Category: database

3+a| #### Type: creation

| Action
| Outcome
| Description

| `saved_object_create`
| `unknown`
| User is creating a saved object.

...
```